yw_website/yw_website/apps/website/views.py

```diff
@@ -177,7 +177,8 @@ def run_detail(request, run_id):
 def delete_workflows(request, workflow_id):
     workflow = get_object_or_404(Workflow, pk=workflow_id)
     workflow.delete()
-    return redirect('my_workflows')
+    return redirect('home')
```

Comment:
i think i like having it redirect to my_workflows instead of home. Do you have a strong preference here?

Reply:
No, likely better that way so it's more obvious it was deleted